download.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/Jguer/yay/v9/pkg/types"
 )
 
+const (
+	gitDiffRefName = "AURUTILS_SEEN"
```

Comment:
Just change this to `AUR_SEEN` and we're good

Reply:
And ditch the unneeded brackets.